donate/settings/development.py

```diff
@@ -40,6 +40,7 @@ def post_setup(cls):
 
 class ThunderbirdDevelopment(Development, ThunderbirdOverrides, Configuration):
     INSTALLED_APPS = ThunderbirdOverrides.INSTALLED_APPS + Development.INSTALLED_APPS
+    ENABLE_THUNDERBIRD_REDIRECT = False
```

Comment:
do we need this if we're already setting it to `false`  by default in `environment.py`?

Reply:
(also, with every configuration explicitly set to `False`, how would one set it to `True`?)

Reply:
Looking at this again... shouldn't these all be `True` in the TB configurations?